ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/SendTransaction.java

```diff
@@ -136,4 +61,9 @@ private JsonObject transactionWithoutField(final String from) {
     eea_sendTransaction.setId(DEFAULT_ID);
     return eea_sendTransaction;
   }
+
+  private <T> void putValue(
```

Comment:
this looks repeated from a previous class. worth extracting in a JsonTestUtil or similar class?

Reply:
done